pkgs/boto/default.nix

```diff
@@ -0,0 +1,24 @@
+{ pkgs ? import <nixpkgs> {}, ... }:
```

Comment:
btw i think it would be nice to move python stuff to `pkgs/python/{boto,...}` because it's getting crowded